examples/jsm/postprocessing/TAARenderPass.js

```diff
@@ -147,8 +147,8 @@ class TAARenderPass extends SSAARenderPass {
 
 		super.dispose();
 
-		if ( this.sampleRenderTarget !== undefined ) this.sampleRenderTarget.dispose();
-		if ( this.holdRenderTarget !== undefined ) this.holdRenderTarget.dispose();
+		if ( this.sampleRenderTarget ) this.sampleRenderTarget.dispose();
```

Comment:
Do you mind if we just remove the entire line? Since `SSAARenderPass.dispose()` already takes care of `sampleRenderTarget`, it's more clean to just focus on `holdRenderTarget` (which is TAA exclusive).

Reply:
Yeah, that makes sense, I just did the minimal change to make it work.

Reply:
Removed